addons/DistancePlus/ja_distance.lua

```diff
@@ -0,0 +1,110 @@
+-- Automatically generated file: Job Abilities
```

Comment:
That's a lie. This file was not automatically generated :)

I see it's become a trend to manually copy things from the resources into a new file, and I will again advise against it. Instead query the proper resources. It even has a `distance` value already, which it calls `range`, so no need to ask the community to contribute. The only difference is that the `range` parameter is not in game units, but has to be translated by a table. Can read some more about it in a [post about it](http://forums.windower.net/index.php?/topic/412-distance-and-gearswap-question/?p=2332).